test/tests/functional/pbs_hook_alarm_large_multinode_job.py

```diff
@@ -150,9 +151,10 @@ def test_epi_hook(self):
         j.set_attributes(a)
 
         jid = self.server.submit(j)
-
         self.server.expect(JOB, {'job_state': 'R'},
                            jid, max_attempts=15, interval=2)
+        self.server.log_match("dequeuing from", starttime=search_after)
```

Comment:
Two things.  First, the job is going to take 10s to finish, so you might as well put in a 10s sleep (with a logger message above it saying what you're doing).  Second, it isn't necessary with the 10s walltime, but you might set the sleep time of the job to 10s.  PBS will kill the job after 10s, but it might take it a little longer to do so.